### DISCUSSION TITLE
Possible Evolution

### DISCUSSION THREAD

**Discussion by Vince56190** [agronick/aa-torque#49] [Ideas]
The navigation bar between the screens is a bit large. Would it be possible to reduce it and add a strip with date and time or simply a clock gauges?
Possible to add a personal wallpaper?
Could the application keep the maximum gauge values in memory?

**Comment by agronick** [comment]
Date and time is already on the AA UI. I agree that the navigation could be smaller. It also wouldn't be hard to keep min and max in memory. The previous app allowed you to reset the guages by tapping them. I liked having them reset when I navigate away from the page but an option to persist it would be good for people who want it.

**Reply by mkwatchface** [reply]
I think the option for the maximum values to persist is a good one! I am currently breaking in my new car and i don't want to push it over 3000 rpm. AA Torgue is helping me see what was the maximum value. I would be good to persist even if navigating away from the page. This is also helpful for longer trips!
